yarn.lock

```diff
@@ -7021,7 +7034,20 @@ exec-sh@^0.3.2:
   resolved "https://registry.yarnpkg.com/exec-sh/-/exec-sh-0.3.2.tgz#6738de2eb7c8e671d0366aea0b0db8c6f7d7391b"
   integrity sha512-9sLAvzhI5nc8TpuQUh4ahMdCrWT00wPWz7j47/emR5+2qEfoZP5zzUXvx+vdx+H6ohhnsYC31iX04QLYJK8zTg==
 
-execa@^0.6.0:
+execa@^0.10.0:
```

Comment:
I had to merge this by hand but I'm not 100% sure I got it right -- how do I regenerate a `yarn.lock` properly?